src/routes.js

```diff
@@ -88,10 +88,10 @@ module.exports = [
   },
   {
     method: 'GET',
-    path: '/lib/{file*}',
+    path: '/node_modules/{file*}',
     handler: {
       directory: {
-        path: 'lib'
+        path: 'node_modules'
```

Comment:
Hi @des-des, materialize-css is in node_modules folder not in a public, should I move it to the public folder?